Source/OxyPlot.ImageSharp/ImageRenderContext.cs

```diff
@@ -434,34 +434,22 @@ public override void DrawPolygon(IList<ScreenPoint> points, OxyColor fill, OxyCo
         }
 
         /// <inheritdoc/>
-        public override void ResetClip()
+        protected override void ResetClip()
         {
             this.EnsureClippedRegion();
 
             this.clipping = false;
         }
 
         /// <inheritdoc/>
-        public override bool SetClip(OxyRect clippingRectangle)
+        protected override void SetClip(OxyRect clippingRectangle)
```

Comment:
This could be optimised by overriding `Push` and `Pop`, as a `Push` would require ensuring the clipping region, but not blitting back from the image. I don't think this should be a priority for this PR.